io.mpv.Mpv.yml

```diff
@@ -567,6 +554,11 @@ modules:
         #  type: git
         #  tag-pattern: ^([\d.]+)$
 
+      # fix cmake build issue
+      # https://bitbucket.org/multicoreware/x265_git/commits/b354c009a60bcd6d7fc04014e200a1ee9c45c167
+      - type: patch
+        path: patches/x265-cmake_policy.patch
```

Comment:
I think this was supposed to be removed as it's fixed above with build option

Reply:
This patch is still required. It is a bit confusing, but if you check [this diff](https://github.com/flathub/io.mpv.Mpv/compare/9df55797fba718d0da90c9930b4f36a6aa0ce5ed..4cdc5ac57a8d96dfe51342ca262a260d6684246c#diff-44b5e31379ee0fdb2e61ebaf8409856ac45f5a15ec3e6a6a2784f5526458dbdfL577-R560), you'll see that there were two patches before, one for the required minimum cmake version (that one was removed) and one for the policy issue. The policy patch is the same that is used [in the freedesktop runtime](https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/92186447b2f86d0ce15552e343c3dd387bed766d/patches/libx265/cmakev4.patch).